apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/ConfirmAlert.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useMemo } from "react";
+import { TFunction } from "i18next";
+import { Trans, withTranslation } from "react-i18next";
+import { Transaction } from "@ledgerhq/live-common/generated/types";
+import { DeviceTransactionField } from "@ledgerhq/live-common/transaction/index";
+import Alert from "~/renderer/components/Alert";
+
+type Props = {
+  t: TFunction;
+  transaction: Transaction;
+  typeTransaction: string | undefined;
+  fields: DeviceTransactionField[];
+};
+
+const ConfirmAlert = ({ t, transaction, typeTransaction, fields }: Props) => {
+  const key = ("mode" in transaction && transaction.mode) || "send";
+  const recipientWording = t(`TransactionConfirm.recipientWording.${key}`);
+
+  const amountTransaction: string = useMemo(
+    () =>
+      (
+        fields.find(
+          (field: { label: string }) => field.label && field.label === "Amount",
+        ) as DeviceTransactionField & { value: string }
```

Comment:
Do we really need the `as` here ?
I suppose it's copied from the other component so we might want to look at that

Reply:
We don't 👍 , cleaned up the memoized function also to make it a bit more readable

Reply:
🧹 https://github.com/LedgerHQ/ledger-live/pull/6205/commits/7082400185baebf65ec655faed2bba8bef4f0f1e